### PR TITLE
Add Organization Administrator to the glossary

### PR DESCRIPTION
**Pull request by spank50**

### CODE REVIEW

**Review by ndeevy — APPROVED** [review]

**Comment by ndeevy** [comment]
LGreatTM @spank50 :-]

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc [hunk 1]
```diff
@@ -164,6 +164,20 @@
 
 *See also*:
 
+[discrete]
+[[organization-administrator]]
+==== image:images/yes.png[yes] Organization Administrator (noun)
+*Description*: From https://access.redhat.com/articles/1757953[Roles and Permissions for Red Hat Customer Portal]: Organization Administrator: This is the highest permission level with full access to content and features. This is the only role that can manage users and control their access and permissions on an account.
```
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc:170]: Even though you do provide that link, I'd consider incorporating that it's the highest permission level for Red Hat accounts

```suggestion
*Description*: From https://access.redhat.com/articles/1757953[Roles and Permissions for Red Hat Customer Portal]: Organization Administrator: This is the highest permission level for a Red Hat account with full access to content and features. This is the only role that can manage users and control their access and permissions on an account.
```

**Review by bergerhoffer — DISMISSED** [review]
One suggestion, but it's minor so I'm still giving my ack even without it!

**Review by ndeevy — APPROVED** [review]

**Review by inoxx03 — APPROVED** [review]

**Comment by inoxx03** [comment]
@redhat-documentation/ccs-style-council We're missing 1 more ACK on this PR. Could someone please review and provide one? Thank you:)

**Comment by cbudz** [comment]
Ack. One question for the future - do we want to allow links to kbase articles? They can be quite fickle in their maintenance.

**Pull request merged**